app/controllers/my_modules_controller.rb

```diff
@@ -534,6 +534,8 @@ def log_user_designation_activity
     users = User.where(id: params[:my_module][:user_ids])
```

Comment:
users = User.where.not(id: current_user.id).where(id: params[:my_module][:user_ids])